src/components/ProofOfAddress/PoAIntro/index.tsx

```diff
@@ -7,6 +7,18 @@ import { localised } from '~locales'
 import { POA_INTRO_LOCALES_MAPPING } from '~utils/localesMapping'
 import theme from '../../Theme/style.scss'
 import style from './style.scss'
+import { WithLocalisedProps } from '~types/hocs'
+import { StepComponentFaceProps } from '~types/routers'
+import { PoaIntroTypes } from '~types/steps'
+
+type PaOIntroProps = { country: string } & WithLocalisedProps &
+  StepComponentFaceProps
+
+const poaIntroTypes: PoaIntroTypes[] = [
+  'shows_address',
+  'matches_signup',
+  'most_recent',
+]
```

Comment:
Why are these changes in this PR?

Reply:
I migrated too much. I move this in an other branch.